Added remaining GLUE tasks to 'run_classifier.py'

ananyahjha93 opened the pull request:
Also added metrics used in the GLUE paper for each task.

thomwolf commented:
Hi @ananyahjha93,
Thanks for this PR.
Do you have some results from fine-tuning BERT on the other tasks?
Also, I think we should add some details on the available tasks in the readme as well.

ananyahjha93 commented:
@thomwolf I have added results on GLUE dev set in the README and details on how to run any GLUE task. But, I have also added a warning stating that all GLUE tasks have not been tested with half-precision training. With the new tasks and metrics being added, there should be one round of tests with half-precision training as well. Unfortunately, I do not have direct access to a V100 or any of the RTX cards in order to run that. 

Also, for some reason the previous pull request was passing each build_py2 test but now it is failing the 'OpenAIGPTTokenizationTest'. I only added changes to the README in my latest commit.

thomwolf commented:
This looks great, thanks @ananyahjha93!
